extensions/eclipse-che-theia-plugin-ext/src/common/che-protocol.ts

```diff
@@ -41,6 +41,19 @@ export interface CheFactoryMain {
     $getFactoryById(factoryId: string): Promise<cheApi.factory.Factory>;
 }
 
+
+export interface CheSsh {
```

Comment:
hello, why do we need this empty interface ?

Reply:
actually i don't know, i did it similar to Factory https://github.com/eclipse/che-theia/blob/master/extensions/eclipse-che-theia-plugin-ext/src/common/che-protocol.ts#L37

Reply:
OK it's a marker interface.